test/test_ecr_utils.py

```diff
@@ -55,6 +58,73 @@ def test_ecr_utils_workflow():
             # XXX: integrated test the remaining?
 
 
-def test_ecr_utils_integrated():
-    """ Write me! """
-    pass
+def test_ecr_utils_get_uri():
+    print()  # start output on fresh line
```

Comment:
Is there a reason for `print` vs. `PRINT` here?

Reply:
Not in tests. Tests just put out debugging info. My test for stray print statements doesn't look in test files.